.pyproject_generation/pyproject_custom.toml

```diff
@@ -1,12 +1,12 @@
 [project]
 name = "mass"
-version = "2.0.0"
+version = "2.0.1"
 description = "Metadata Artifact Search Service  - A service for searching metadata artifacts and filtering results."
 dependencies = [
     "typer>=0.9.0",
```

Comment:
Other services/tools have the lower bound as 0.12

Reply:
Ah, didn't notice. I'll update it